docs/src/reference-guide/plugins/forc-index/index.md

```diff
@@ -22,5 +22,6 @@ SUBCOMMANDS:
     init      Create a new indexer project in the current directory
     new       Create a new indexer project in a new directory
     remove    Stop and remove a running index
+    revert    Revert a running index to the last deployed version
```

Comment:
I should probably clarify some terminology at our next team sync as I think I started this trend of calling indexers indices 😅 
```suggestion
    revert    Revert a running indexer to its last previous version
```